args/args_test.go

```diff
@@ -126,7 +126,7 @@ func TestEnvironmentVarsOverrideCliArgs(t *testing.T) {
 	clearFlagSet()
 	assert.NoError(t, sdk_args.SetupArgs(&args))
 
-	assert.False(t, args.Verbose)
+	assert.True(t, args.Verbose)
```

Comment:
The name of the test should change to `TestCliArgsOverrideEnvironmentArgs`.
As "meta-test" I would also set an environment flag different than the default one, without CLI, and see that in this case, the Environment var has been successfully used.

Reply:
So true, I've changed the test name, regarding the environment vars there is another test already in:  https://github.com/newrelic/infra-integrations-sdk/blob/master/args/args_test.go#L84